backend/database/versions/fa0950f6df19_add_summary_link.py

```diff
@@ -0,0 +1,57 @@
+"""add_summary_link
+
+Revision ID: fa0950f6df19
+Revises: 7968f161b60c
+Create Date: 2020-08-24 11:04:30.640496
+
+"""
+from alembic import op
+from sqlalchemy.dialects.postgresql import ENUM
+
+
+# revision identifiers, used by Alembic.
+revision = "fa0950f6df19"
+down_revision = "7968f161b60c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    project_link_enum = ENUM(name="projectlink")
```

Comment:
where does name="projectlink" come from? I don't see it set in corpora_orm.py. Should we be setting the name in corpora_orm.py as well?

Reply:
Afaict, it's not related to the orm code except that there are identical python enums in there. Searching around a little, it does seem like there are tools or patterns [[1](https://pypi.org/project/SqlAlchemy-Enum-Tables/#description), [2](https://www.thetopsites.net/article/53556214.shtml)] to tighten the link between them. That said, the migration script is just run once.